src/main/java/duke/DukeException.java

```diff
@@ -0,0 +1,16 @@
+package duke;
+
+/**
+ * For exceptions that happen while running the Duke.
+ */
+public class DukeException extends Exception {
```

Comment:
Perhaps you could have exceptions extending from DukeException? (Such as DukeInvalidDateFormatException). This may help to distinguish between different types of DukeExceptions thrown.